tools/serverpod_cli/lib/src/commands/cloud/logout.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:serverpod_cli/src/downloads/resource_manager.dart';
+import 'package:serverpod_cli/src/logger/logger.dart';
+import 'package:serverpod_cli/src/runner/serverpod_command.dart';
+import 'package:serverpod_cli/src/util/exit_exception.dart';
+
+class CloudLogoutCommand extends ServerpodCommand {
+  @override
+  final name = 'logout';
+
+  @override
+  final description = 'Log out from the Serverpod cloud.';
+
+  @override
+  void run() async {
+    var cloudData = await resourceManager.tryFetchServerpodCloudData();
+
+    if (cloudData == null) {
+      log.info('You are not logged in to the Serverpod cloud.');
+      return;
+    }
+
+    // TODO: Invalidate the token on the server.
```

Comment:
Question: Do we have a logout endpoint in the auth module? If so I think we should add this now already and deal with the consequences for the tests.

Reply:
We do, but since we have not published the client yet I didn't want to depend upon it. I will add this once the client is published.